tests/devices/test_default_qubit_jax.py

```diff
@@ -439,11 +440,13 @@ def test_non_backprop_error(self):
         dev = qml.device("default.qubit.jax", wires=2)
 
         def circuit(weights):
-            qml.templates.StronglyEntanglingLayers(weights, wires=[0, 1])
+            qml.RX(weights[0], wires=0)
+            qml.RY(weights[1], wires=1)
+            qml.CNOT(wires=[0, 1])
             return qml.expval(qml.PauliZ(0))
 
         qnode = qml.QNode(circuit, dev, interface="jax", diff_method="parameter-shift")
-        weights = jnp.array(qml.init.strong_ent_layers_normal(n_wires=2, n_layers=2))

```

Comment:
Interesting. Why this change? Just to remove a dependency?

Reply:
Yep, this test is testing JAX differentiation, so shouldn't need to depend on extra template code. This narrows the test to ensure that it passes if JAX differentiation works as intended. This way, if the test fails, we know for sure why (as opposed to it potentially being a template issue).

However, we should add a separate test to ensure that _templates_ can be differentiated.